packages/perspective-viewer-d3fc/src/js/charts/ohlcCandle.js

```diff
@@ -90,7 +95,8 @@ function ohlcCandle(seriesSvg) {
             .chart(chart)
             .settings(settings)
             .xScale(xScale)
-            .yScale(yScale);
+            .yScale(yScale)
+            .canvas(true);
 
         const toolTip = nearbyTip()
             .settings(settings)
```

Comment:
```suggestion
            .settings(settings)
            .canvas(true)
```

Reply:
I think this is needed to make tooltips work

Reply:
I don't know why I didn't check tooltips but I've confused by this suggestion.

Reply:
I'm not sure why it isn't working

Reply:
I see, it's the `canvas` property that needs to be set.

Reply:
I think you just need to add .canvas(true). It worked for me:

const toolTip = nearbyTip()
            .settings(settings)
            .xScale(xScale)
            .yScale(yScale)
            .yValueName("closeValue")
            .canvas(true)
            .data(data);